projects/Monaliza.md

```diff
@@ -0,0 +1,90 @@
+## Project Name 
+
+Monaliza
+
+## Category 
+
+NFT, Truffle, NodeJS, React, Python, IPFS Storage, CELO, Docker, Kubernetes, Solidity
+
+## Project Description
+
+The project is aimed to be a connector between instagram/social media and NFTs, enabling the average instagram/social media user to be able to mint their own NFTs and submit them straight to their instagram, utilising their fanbase and followers, as well as Opensea, and persist this to IPFS as a storage point. This would hopefully get the average person to be able to adopt blockchain without having to leave the comfort of their favourite platform and make their instagram posts/pictures/videos an NFT.
+
+## Project Status
+
+Under Development
+
+## Previews
+
+![Landing Page Preview](/home/darien/Downloads/MonaSS.png)
+
+## Target Audience
+
+Our target audience is those already using social media as a means of displaying their art. 
+Artists, Animators, Illustrators, Photographers and the likes.
+
+## Rough estimated user base (if applicable)
+
+We're hoping both the users of the platform will be able to tap into their fanbase as well as users of social media platforms.
+
+## Github repo
+
+https://github.com/monalisanft/nfthack
+
+## Website
+
+Landing Page - https://monaliza.app/
+
+## Docs
+
+https://www.notion.so/monalisanft/Monaliza-a624fe444b11470db6972bdc134984f1
+
+## Team Info
+
+Aina
+- Project Manager
+- Background In Corporate Finance
+- Entrepreneur in Fintech
+- Consultant
+- Blockchain Professor in a Business School
+
+Seema
+- Blockchain Backend Developer with a focus on Solidity, Truffle, NodeJS and Infrastructure as Code
+
+Jasmin
+- Marketing
+- Curator of all things Art
+- Entrepreneur
+
+George
+- Solutions Architect
+
+Darien
+- Security Engineer
+- Front End/DApp Developer
+- Crypto/Blockchain Specialist
+
+### Team Size  
+
+Currently a team of 5
+
+### Team members  
+
+Darien Ford
+Jasmin Fiori
+George Clarke
+Aina Raherimanantsoa
+Seema Sharma
+
+## How the community can engage
+GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->  

```

Comment:
Apologies! i missed this earlier - please create the show-and-tell thread per the instructions here: https://github.com/ipfs/community#ecosystem-projects

Reply:
No worries, I believe I've done this too. https://github.com/ipfs/community/discussions/493

Reply:
Yes, but please add the link here.

Reply:
Hi, I've just added the link